x/staking/README.md

```diff
@@ -874,7 +874,6 @@ The staking module emits the following events:
 | Type           | Attribute Key       | Attribute Value     |
```

Comment:
this readme's missing all the LSM transactions and parameters

Reply:
True. I couldn't find an updated version of this doc in [SDK v0.45.16-ics-lsm](https://github.com/cosmos/cosmos-sdk/tree/v0.45.16-ics-lsm) either. Do you know if it exists?

Reply:
I don't think it exists unfortunately - I don't think x/staking even had a README in v0.45

[These](https://github.com/cosmos/cosmos-sdk/blob/v0.45.16-ics-lsm/docs/core/proto-docs.md) were the only docs I was aware of (auto generated proto docs)

Reply:
Doc updated [here](d3f2a80684a62b71ce6258b3630fecb991a95a02)!

Reply:
looks great!